src/search.ts

```diff
@@ -81,7 +81,8 @@ export async function searchIndex(config: Config, effects = defaultEffects): Pro
         options: {
           fields: indexOptions.fields,
           storeFields: indexOptions.storeFields
-        }
+        },
+        cleanUrls: config.md.normalizeLink("./hello") === "./hello"
```

Comment:
Similar to above. If we expect `md.normalizeLink` to be overridden and have some other behavior, then we need to pass the normalized links to minisearch so that it renders normalized links on the client. On the other hand, if we only want to support our more limited definition of `cleanUrls`, then we shouldn’t test the behavior of `md.normalizeLink`; we should explicitly track the setting in the config.

I guess it’d be cool to respect the implementation of `md.normalizeLink`, if it’s not too much work. Perhaps we could invoke it up above when computing the `id` rather than having the client do the normalization and repeating the logic?